src/collectors/kafkastat/kafkastat.py

```diff
@@ -132,19 +126,17 @@ def query_mbean(self, objectname, key_prefix=None):
         for attrib in attributes.getiterator(tag='Attribute'):
             atype = attrib.get('type')
 
-            ptype = self.ATTRIBUTE_TYPES.get(atype)
-            if not ptype:
+            try:
+                value = float(attrib.get('value'))
+            except:
```

Comment:
Don't use a bare `except`, this can cause unwanted side effects. Is there a reason these needs to be more than `except (ValueError, TypeError)` ?